### ISSUE TITLE
"Persistent datasets" term is confusing. Consider changing it into "Unmanaged datasets"

### ISSUE THREAD

**Issue by lukasz-zaroda** [ubuntu/zsys#200]
**Describe the bug**
`Persistency` suggest some kind of a feature, but it seems that it's rather lack of features. Datasets outside of the namespace are simply not managed by Zsys.

**To Reproduce**
1. Read https://didrocks.fr/2020/06/16/zfs-focus-on-ubuntu-20.04-lts-zsys-dataset-layout/
2. By confused by description of persistent datasets.

**Expected behavior**
User should understand what persistent datasets are without paragraphs of explanation. Term "Unmanaged datasets" is mostly self explanatory, or at least provides the better starting point in understanding them as featureless datasets, and not ones with some `persistent` feature added by Zsys.

**Comment by almereyda** [comment]
Yes, I agree very much with the notion that `zsys` cannot make any assumptions about non-managed datasets.

Similarily to what is expressed in

* [backup - How does one exclude a zpool from zfs-auto-snapshot - Ask Ubuntu](https://askubuntu.com/a/1119949/220802)

for `zfs-auto-snapshot --default-exclude`, `zsys` should only consider to manage datasets that have a special metadata entry set, and not only infer its role by the namespace of a dataset, and eventual snapshot names. This anti-pattern could be read in similarity to how [zfsnap/zfsnap: A portable, performant script to make rolling ZFS snapshots easy. (https://github.com/zfsnap/zfsnap/issues/109)](https://github.com/zfsnap/zfsnap) worked:

> zfsnap stores all the information it needs about a snapshot directly in its name; no database or special ZFS properties are needed. The information is stored in a way that is human readable, making it much easier for a sysadmin to manage and audit backup schedules.
>
> Snapshot names are in the format of pool/fs@[prefix]Timestamp--TimeToLive (e.g. pool/fs@weekly-2014-04-07_05.30.00--6m). The prefix is optional but can be quite useful for filtering, Timestamp is the date and time when the snapshot was created, and TimeToLive (TTL) is the amount of time the snapshot will be kept until it can be deleted.

Coming to a shared understanding within the `zsys` maintainers and the affected, distinct cohort of its users could help mitigating other issues, too:

* https://github.com/ubuntu/zsys/issues/196
* https://github.com/ubuntu/zsys/issues/203

- https://github.com/moby/moby/issues/41055
